mixer/template/edge/template.proto

```diff
@@ -14,6 +14,12 @@
 
 syntax = "proto3";
 
+// $title: Edge
+// $description: A template designed to report observed communication edges between workloads.
+// $location: https://istio.io/docs/reference/config/policy-and-telemetry/templates/edge.html
+// $aliases:
```

Comment:
You don't need the $aliases and subsequent line, that's only used when moving things to new locations. SInce this is all new, no need.

Reply:
Removed.